R/build-quarto-articles.R

```diff
@@ -96,14 +108,19 @@ quarto_render <- function(pkg, path, quiet = TRUE, frame = caller_env()) {
   write_yaml(quarto_format(pkg), metadata_path)
 
   output_dir <- withr::local_tempdir("pkgdown-quarto-", .local_envir = frame)
-  quarto::quarto_render(
-    path,
-    metadata_file = metadata_path,
-    execute_dir = output_dir,
-    quarto_args = c("--output-dir", output_dir),
-    quiet = quiet,
-    as_job = FALSE
-  )
+
+  # set the lib path to the temporary pkgdown installation directory
+  lib_path <- .libPaths()[1]
```

Comment:
this is the meaningful change.

air reformatted this file so there are a bunch of unrelated changes.